stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -862,6 +862,13 @@ const CHAINSTATE_SCHEMA_4: &[&str] = &[
     "#,
 ];
 
+const CHAINSTATE_SCHEMA_5: &[&str] = &[
```

Comment:
Who came up with this naming scheme 😰? Following the existing naming, maybe this should be `NAKAMOTO_CHAINSTATE_SCHEMA_6` and be added in stackslib/src/chainstate/nakamoto/mod.rs instead?